webapp/app.py

```diff
@@ -121,6 +121,7 @@ def dashboard_index(path=None):
 
         return flask.render_template("dashboard/index.html")
 
+    @app.route("/config.js")
```

Comment:
If this is how you want to be able to access `/config.js`, then you should be copying `config.js` into `templates/dashboard/.`, rather than `static/.`.

The alternative is to copy it into `static`, as you are, but then changing any front-end thing that wants to download it to use the URL `/static/config.js` instead.

Reply:
Solved